src/components/CV_hierarchy/hierarchy.js

```diff
@@ -184,11 +187,17 @@ function CustomizedTreeView(props) {
   };
 
   const handleSelect = (event, nodeIds) => {
+    // nodeIds[0] is the selected label 
     const difference = nodeIds
       .filter(x => !value.includes(x))
       .concat(value.filter(x => !nodeIds.includes(x)));
-    setSelected(nodeIds);
-    onClick(difference[0])
+    if (value.includes(nodeIds[0])){
```

Comment:
select option can be either selecting a new node / or un-selecting a previous selected node.